public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -82,6 +120,43 @@ function Keyboard(menuUI) {
         }
     }
 
+    function _unsureShortcutPressed() {
+        status.unsureSelected = true;
+        status.disagreeSelected = false;
+        if (svv.newValidateBeta) {
+            svv.ui.newValidateBeta.unsureButton.click();
+        } else {
+            let comment = menuUI.comment.val();
+            validateLabel(menuUI.unsureButton, "Unsure", comment);
+            menuUI.yesButton.removeClass("validate");
+            menuUI.noButton.removeClass("validate");
+        }
+    }
+
+    function _setSeverity(severity){
+        if(svv.newValidateBeta){
+            svv.ui.newValidateBeta.severityMenu.find(`#severity-button-${severity}`).click();
+        } else {
```

Comment:
`menuUI.severityMenu` doesn't exist if `svv.newValidateBeta` is `false`...

Reply:
done